sunpy/visualization/animator/wcs.py

```diff
@@ -299,10 +306,27 @@ def plot_start_image_2d(self, ax):
 
         return im
 
+    def _get_2d_plot_limits(self):
+        """
+        Get vmin, vmax of a data slice when clip_interval is specified.
+        """
+        if len(self.clip_interval) == 2:
```

Comment:
I wonder if this check is better to happen in the init instead. I don't think we should allow a bad clip_interval to be passed in and return the class.

Reply:
Okay, I'll make this check within init itself.